test/integration/smart_answer_flows/overseas_passports_test.rb

```diff
@@ -289,16 +244,12 @@ class OverseasPassportsTest < ActiveSupport::TestCase
       add_response 'renewing_new'
       add_response 'adult'
       assert_current_node :ips_application_result_online
-      assert_phrase_list :how_long_it_takes, [:how_long_6_weeks, :how_long_additional_info_renewing_new, :how_long_additional_time_online]
-      assert_phrase_list :cost, [:passport_courier_costs_ips1, :adult_passport_costs_ips1]
-      assert_phrase_list :how_to_apply, [:how_to_apply_online, :how_to_apply_online_prerequisites_renewing, :how_to_apply_online_guidance_doc_group_3, :how_to_apply_online_guidance_doc_outro]
     end
     should "use the document group of the country of birth - Spain (which is 1)" do
```

Comment:
Yes. Initially when doing these conversions I tried to change the tests to keep them making sense, but having chatted to @chrisroos about it we decided that since we imagine these tests going away that it probably wasn't worth the time it would take. So since then I've been doing the bare minimum to keep the tests passing which mainly means removing `assert_phrase_list` statements.